core/services/ocr2/plugins/ocr2vrf/coordinator/coordinator.go

```diff
@@ -35,6 +37,41 @@ var (
 	dkgABI            = evmtypes.MustGetABI(dkg_wrapper.DKGMetaData.ABI)
 	vrfBeaconABI      = evmtypes.MustGetABI(vrf_beacon.VRFBeaconMetaData.ABI)
 	vrfCoordinatorABI = evmtypes.MustGetABI(vrf_coordinator.VRFCoordinatorMetaData.ABI)
+	buckets           = []float64{
+		0,
+		1,
+		2,
+		4,
+		8,
+		16,
+		32,
+		64,
+		128,
+		256,
+		512,
+		1024,
+		2048,
+	}
+	promBlocksToReport = promauto.NewHistogramVec(prometheus.HistogramOpts{
+		Name:    "ocr2vrf_coordinator_blocks_to_report",
+		Help:    "Number of unfulfilled blocks in ReportBlocks",
```

Comment:
nit: it's not just the "unfulfilled blocks" or callbacks, it's the unfulfilled or inflight blocks that fit in the current batch.

Reply:
true 👍